gradio/themes/default.py

```diff
@@ -47,6 +47,8 @@ def __init__(
         super().set(
             # Colors
             input_background_fill_dark="*neutral_800",
+            error_background_fill=colors.red.c200,
+            error_border_color=colors.red.c600,
```

Comment:
In order to affect the other themes, should we add these changes to `base.py` instead?